### PR TITLE
Contracts list v3

### PR DESCRIPTION
**Pull request by goranradmanovic**
@vgkids Please check this new PR for Contract List page

### CODE REVIEW

File: src/views/ContractsView.vue [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    apps.forEach(async app => {
+      let totalErrors = 0
+
+      try {
+        app.stats = await store.state.Session.apiCall(`/activity_entries/stats?app_id=${app.name}`)
+
+        app.stats.forEach(item => {
+          item.hasErrors = item?.count.hasOwnProperty('500') ? true : false
+          totalErrors += item?.count['500'] || 0
+          item.barHeight = ((item?.count['500'] || 0) + (item?.count['200'] || 0)) || 1 // Default bar chart hight is 1px
```
Comment (vgkids) [inline src/views/ContractsView.vue:210]: We want the bar height to be proportional for the total amount of activity.

Here's a method that gives you the total amount of activity from the `/stats/` response, and also its proportional bar height. The day with the highest activity will have `proportionalHeight: 100`.

If there are errors, the bar height should still be the same, but the bar will be red.

``` javascript
const addProportionalHeightAndTotal = (data) => {
  // Calculate total count for each day and find the maximum
  const maxTotal = data.reduce((max, item) => {
    const total = Object.values(item.count).reduce((sum, value) => sum + value, 0);
    return Math.max(max, total);
  }, 0);

  // Add proportional height and total to each item
  return data.map(item => {
    const total = Object.values(item.count).reduce((sum, value) => sum + value, 0);
    const proportionalHeight = Math.round((total / maxTotal) * 100);
    return {
      ...item,
      count: {
        ...item.count,
        total
      },
      proportionalHeight
    };
  });
};
```

**Review by vgkids — CHANGES_REQUESTED** [review]

**Comment by vgkids** [comment]
Full list:

- bar height should be proportional to total activity
- the table is scrolling inside a container, even though there's plenty of page unused
- the tooltip is obscured by whatever this inner container is
Screenshot 2024-09-11 at 12 37 40 PM
- Let's add this Filter / Search bar from the existing contracts page using the same matching, it is very helpful for a quick narrowing of results, without having to use the column filters.
Screenshot 2024-09-11 at 12 40 50 PM

**Comment by goranradmanovic** [comment]
@vgkids Please check new changes on this PR based on your last commets and loom video

**Review by vgkids — APPROVED** [review]
Looks great. I made the links in the sidebar and breadcrumb point to the new path, and squashed @goranradmanovic's commits into one.

We'll want to re-visit the path name later; the quick `add-a-3-on-the-end` versioning here (done at my request) is fast but will catch up with us eventually.

**Pull request merged**